src/analysis/utils/disk-utils.ts

```diff
@@ -37,18 +37,11 @@ export const pdTypeFromDiskType = (type: GoogleDiskType): GooglePdType =>
      */
   ) as GooglePdType; // TODO: Remove cast
 
-export const updatePdType = (disk: PersistentDisk): DecoratedPersistentDisk => ({
+export const updatePdType = (disk: RawPersistentDisk): PersistentDisk => ({
```

Comment:
nit: for naming, this could be treated as a black box that takes a RawPD and returns a PD - it could be called `toPersistentDisk` or `asPersistentDisk`